packages/expect/src/__tests__/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../../../../tsconfig.test.json",
+  "compilerOptions": {
+    "lib": ["es2022"]
```

Comment:
Instead of the entire `2022`, just pull in what's needed for `Error.prototype.cause`

Reply:
I tried some patterns.
```
    "lib": ["es2020","es2022.error", "dom"],

    "lib": ["es2022.error", "es2020", "dom"],

    "lib": ["es2022.error", "dom"],
```

However, the following errors occured in compilation.
https://github.com/facebook/jest/actions/runs/3470037100/jobs/5797753863

```
Error: node_modules/typescript/lib/lib.es2022.error.d.ts(69,8): error TS2304: Cannot find name 'AggregateError'.
Error: node_modules/typescript/lib/lib.es2022.error.d.ts(74,8): error TS2304: Cannot find name 'AggregateError'.
```

I investigated but could not find the cause.